### ISSUE TITLE
What is the output of the below code?

### ISSUE THREAD

**Issue by AshishNamdev** [AshishNamdev/iqa#21]
What is the output of the below code?
```python
>> import array
>>> a = [1, 2, 3]
>>> print a[-3]
>>> print a[-2]
>>> print a[-1]
```
### Output:
```bash
3, 2, 1
```

**Comment by AshishNamdev** [comment]
Included in [Q8](../blob/master/questions/Q1-50.md#Q8)

**Comment by AshishNamdev** [comment]
included in commit 0ee53c12045d5ea4788e9b4527a052dd89c4cfdd
